### PR TITLE
SAA-1412 enable offender merge updates in preprod and prod.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
Also improving indexing on the local audit table which the offender merge process will use when retrieving and updating prisoner audit records.

### CODE REVIEW

File: src/main/resources/migrations/common/V2023.12.15__add_index_to_local_audit.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS idx_prison_code_prisoner_number on local_audit(prison_code, prisoner_number);
```
Comment (matt-crowson-moj) [inline src/main/resources/migrations/common/V2023.12.15__add_index_to_local_audit.sql:1]: Also included as noticed missing indexes of audit table which will be hit by the offender merge queries.

Comment (timharrison-moj) [reply src/main/resources/migrations/common/V2023.12.15__add_index_to_local_audit.sql:1]: Are these unique names, Matt? I usually create with the table and column names combined to make each unique.

**Review by timharrison-moj — DISMISSED** [review]
LGTM

File: src/main/resources/migrations/common/V2023.12.15__add_index_to_local_audit.sql (hunk 1 above)
Comment (matt-crowson-moj) [reply src/main/resources/migrations/common/V2023.12.15__add_index_to_local_audit.sql:1]: Will change, yes unique as ran locally.

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**